packages/portal/src/Portal/Portal.tsx

```diff
@@ -15,39 +18,52 @@ export function usePortalContainer(customContainer?: HTMLElement) {
 
   useIsomorphicLayoutEffect(() => {
     if (customContainer) {
+      if (portalRef) {
+        portalRef.current = customContainer;
```

Comment:
Can we add some comments to clarify what's going on here. It's definitely not typical 🙃